src/install/semver.zig

```diff
@@ -1106,70 +1130,67 @@ pub const Range = struct {
     /// >= x
     /// >= 0
     pub fn anyRangeSatisfies(this: *const Range) bool {
-        return this.left.op == .gte and this.left.version.eql(Version{});
+        return this.left.op == .gte and this.left.version.eql(.{});
     }
 
     pub fn initWildcard(version: Version, wildcard: Query.Token.Wildcard) Range {
         switch (wildcard) {
             .none => {
-                return Range{
-                    .left = Comparator{
+                return .{
+                    .left = .{
                         .op = Op.eql,
                         .version = version,
                     },
                 };
             },
 
             .major => {
-                return Range{
-                    .left = Comparator{
+                return .{
+                    .left = .{
                         .op = Op.gte,
-                        .version = Version{
+                        .version = .{
                             // .raw = version.raw
                         },
                     },
                 };
             },
             .minor => {
-                var lhs = Version{
+                const lhs = Version{
+                    .major = version.major + 1,
                     // .raw = version.raw
                 };
-                lhs.major = version.major + 1;
-
-                var rhs = Version{
+                const rhs = Version{
+                    .major = version.major,
                     // .raw = version.raw
                 };
-                rhs.major = version.major;
-
-                return Range{
-                    .left = Comparator{
+                return .{
+                    .left = .{
                         .op = Op.lt,
                         .version = lhs,
                     },
-                    .right = Comparator{
+                    .right = .{
                         .op = Op.gte,
                         .version = rhs,
                     },
                 };
             },
             .patch => {
-                var lhs = Version{};
-                lhs.major = version.major;
-                lhs.minor = version.minor + 1;
-
-                var rhs = Version{};
-                rhs.major = version.major;
-                rhs.minor = version.minor;
-
-                // rhs.raw = version.raw;
-                // lhs.raw = version.raw;
-
+                const lhs = Version{
+                    .major = version.major,
+                    .minor = version.minor + 1,
```

Comment:
pedantic:

```suggestion
                    .minor = version.minor +| 1,
```

Reply:
Done in https://github.com/oven-sh/bun/pull/1854/commits/f0e686aa6b42639a78f758b9a9c76b6d046d1d85